ONSdigital/eq-questionnaire-runner
# Add calculated summary negative demo

### What is the context of this PR?

This PR demonstrates that Runner is able to handle negative values. Within the calculated summary schema, a minimum value was added to each of the answers to provide compatibility for handling this.

### How to review

- Run functional tests to ensure all they pass
- Check the changes in the schema


### Checklist
